exercises/practice/minesweeper/.meta/config.json

```diff
@@ -1,6 +1,59 @@
 {
   "blurb": "Add the numbers to a minesweeper board",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/dcfe347db2c3685954439a72694958dcfe94a3b0